### ISSUE TITLE
Sway reload crashes Firefox-nightly

### ISSUE THREAD

**Issue by J0nnyMak0** [swaywm/sway#5581]
When I do `swaymsg reload` firefox-nightly (running wayland native) crashes. Nothing in sway.log. Using `sway version 1.5-b20d52f7 (Jul 28 2020, branch 'master')`

**Comment by J0nnyMak0** [comment]
from dmesg:
```
[  537.576597] Web Content[7468]: segfault at 0 ip 00007f1bbbb97a50 sp 00007ffe442f1ad8 error 6 in libxkbcommon.so.0.0.0[7f1bbbb8f000+1b000]
[  537.576605] Code: 2e 0f 1f 84 00 00 00 00 00 90 48 85 f6 48 8d 05 e6 89 ff ff 48 0f 44 f0 48 89 77 08 c3 66 66 2e 0f 1f 84 00 00 00 00 00 66 90 <83> 07 01 48 89 f8 c3 66 0f 1f 84 00 00 00 00 00 41 55 48 89 f2 49
[  537.576736] audit: type=1701 audit(1595955626.372:130): auid=1000 uid=1000 gid=1000 ses=1 pid=7468 comm=57656220436F6E74656E74 exe="/home/jonm/usr/local/share/software/firefox/firefox-bin" sig=11 res=1
```

**Comment by Geo25rey** [comment]
It seems to be an I/O error on /usr/share/X11/xkb. I would assume multiple threads or processes of sway are trying to read from it. But keep in mind, spamming reload doesn't really fall under typical use

**Comment by J0nnyMak0** [comment]
> But keep in mind, spamming reload doesn't really fall under typical use

Not sure what you mean by spamming reload. I'm only issuing `swaymsg reload' once.

**Comment by Geo25rey** [comment]
I was only able to replicate the by hitting my reload keybind (defined in my config) in quick succession. Note: if I run `while true; do swaymsg reload; done` in the terminal it does nothing while firefox remains opened. This may be a bug with libinput, but I'm not 100% sure of the architecture. It could also be a bug with xkbcommon.

**Comment by TheAvidDev** [comment]
Does this happen with other apps? If not, have you reported this to Firefox?

> Problems with the Wayland version of Firefox are likely to be Firefox bugs. Start by submitting your issue to the Firefox Bugzilla and come back here only after they confirm otherwise.

**Comment by emersion** [comment]
Sounds like a Firefox bug.

**Comment by travankor** [comment]
duplicate of https://github.com/swaywm/wlroots/issues/2334

**Comment by emersion** [comment]
@travankor Nope, there is no segfault there (a Wayland protocol error instead).

**Comment by fuzunspm** [comment]
When i reload Sway, only Firefox crashes here. All other apps (Emacs, Slack, Zulip, Alacritty, Thunderbird) stays up and running

**Comment by luispabon** [comment]
Probably this https://bugzilla.mozilla.org/show_bug.cgi?id=1652820

**Comment by dzintars** [comment]
Same issue
